storage/driver.go

```diff
@@ -146,3 +147,19 @@ func (v Volume) Validate() error {
 
 	return nil
 }
+
+// Get can be used to get only "string" types from element `driver`
+func (p Params) Get(attr string, allowEmpty bool) (string, error) {
```

Comment:
so if I pass a nil instead of a string by accident and allowEmpty is set, it will be processed as a nil, not a string. Is this the behavior you expect?